tests/integration/kubernetes/tests_common.sh

```diff
@@ -130,6 +130,26 @@ auto_generate_policy_enabled() {
 	[ "${AUTO_GENERATE_POLICY}" == "yes" ]
 }
 
+# adapt common policy settings for tdx
+adapt_common_policy_settings_for_tdx() {
+
+	local settings_dir=$1
```

Comment:
`settings_dir` is unused on this function. Maybe you want to use it instead of `default_genpolicy_settings_dir`?

Reply:
Updated to use correct var. Thanks!